src/deploy/functions/runtimes/discovery/index.ts

```diff
@@ -60,7 +60,7 @@ export async function detectFromPort(
   port: number,
   project: string,
   runtime: runtimes.Runtime,
-  timeout = 10_000 /* 10s to boot up */
+  timeout = 50_000 /* 10s to boot up */
```

Comment:
Can we update the comment to match the updated timeout?

Reply:
Done!